.changes/unreleased/Under the Hood-20230426-140930.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Update version of used Github Actions
+time: 2023-04-26T14:09:30.604603-05:00
+custom:
+  Author: davidbloss
+  Issue: "577"
```

Comment:
You can delete this and then add the `Skip Changelog` label to the PR. Since these changes have 0 effect on end users, we don't need a changelog.